### PR TITLE
[17.0][MIG] account_loan: Migration to 17.0

### PR DESCRIPTION
**Pull request by JT-Odoo**

### CODE REVIEW

**Comment by FernandoRomera** [comment]
@JT-Odoo can you rebase for rebuild runbot?

**Review by victoralmau — CHANGES_REQUESTED** [review]
Please, cherry-pick https://github.com/OCA/account-financial-tools/pull/1952 to commit history and squash administrative commits.

File: account_loan/security/account_loan_security.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="account_loan_multi_company_rule" model="ir.rule">
+        <field name="name">Account loan multi-company</field>
+        <field ref="model_account_loan" name="model_id" />
+        <field eval="True" name="global" />
+        <field name="domain_force">
+            ['|',('company_id','=',False),('company_id','in',company_ids)]
+        </field>
```
Comment (victoralmau) [inline account_loan/security/account_loan_security.xml:9]:
```suggestion
        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
```

File: account_loan/security/account_loan_security.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    </record>
+
+    <record id="account_loan_line_multi_company_rule" model="ir.rule">
+        <field name="name">Account loan line multi-company</field>
+        <field ref="model_account_loan_line" name="model_id" />
+        <field eval="True" name="global" />
+        <field name="domain_force">
+            ['|',('company_id','=',False),('company_id','in',company_ids)]
+        </field>
```
Comment (victoralmau) [inline account_loan/security/account_loan_security.xml:18]:
```suggestion
        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
```

File: account_loan/wizards/account_loan_generate_entries_view.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                    <field name="loan_type" />
+                </group>
+                <footer>
+                    <button
+                        name="run"
+                        string="Run"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
```
Comment (victoralmau) [inline account_loan/wizards/account_loan_generate_entries_view.xml:22]:
```suggestion
```

File: account_loan/wizards/account_loan_increase_amount.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                    <field name="currency_id" />
+                </group>
+                <footer>
+                    <button
+                        name="run"
+                        string="Run"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
```
Comment (victoralmau) [inline account_loan/wizards/account_loan_increase_amount.xml:24]:
```suggestion
```

File: account_loan/wizards/account_loan_pay_amount_view.xml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+                    <field name="currency_id" />
+                </group>
+                <footer>
+                    <button
+                        name="run"
+                        string="Run"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
```
Comment (victoralmau) [inline account_loan/wizards/account_loan_pay_amount_view.xml:26]:
```suggestion
```

File: account_loan/wizards/account_loan_post_view.xml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                    <field name="journal_id" />
+                </group>
+                <footer>
+                    <button
+                        name="run"
+                        string="Run"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
```
Comment (victoralmau) [inline account_loan/wizards/account_loan_post_view.xml:23]:
```suggestion
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by victoralmau** [comment]
Superseded by https://github.com/OCA/account-financial-tools/pull/1960

**Pull request closed without merging**